Makefile

```diff
@@ -89,16 +89,20 @@ uninstall: ## uninstall pouchd and pouch binary
 	@rm -f $(addprefix $(DEST_DIR)/bin/,$(notdir $(DAEMON_BINARY_NAME)))
 	@rm -f $(addprefix $(DEST_DIR)/bin/,$(notdir $(CLI_BINARY_NAME)))
 
-.PHONY: download_dependencies
-download_dependencies: ## download related dependencies, such as dumb_init
+.PHONY: package-dependencies
+package-dependencies: ## install containerd, runc and lxcfs dependencies for packaging
 	@echo $@
-	hack/install/install_ci_related.sh
 	hack/install/install_containerd.sh
+	hack/install/install_lxcfs.sh
+	hack/install/install_runc.sh
+
+.PHONY: download-dependencies
+download-dependencies: package-dependencies ## install dumb-init, local-persist, nsenter and CI tools dependencies
```

Comment:
I think we should change .travis.yml or .circle.yml

Reply:
sure, I have updated .travis.yml with Makefile changes.